rest_test/run_all_tests.sh

```diff
@@ -0,0 +1,109 @@
+#! /bin/bash
```

Comment:
Is there a reason for keeping this file if we have `run_all_tests_from_make.sh`? if not I'd remove it to keep the folder tidy

Reply:
No reason. I will remove.